### DISCUSSION TITLE
blocksuite-react vercel preview removal

### DISCUSSION THREAD

**Discussion by himself65** [toeverything/blocksuite#844] [General]
As in https://github.com/toeverything/blocksuite/issues/818, we want to merge the next code into the `playground` package. However, the next.js build is necessary because we must ensure stability in `SSR` mode in case of a building error in the `AFFiNE` repo.

So the compromise is we build the next in GitHub CI instead of the vercel build. See https://github.com/toeverything/blocksuite/commit/237eccf3b57f03431e5e70e967aaa5060107a5e0. This will cause the next preview to be temporarily unavailable until we really move the code from next to the playground.

/cc @doodlewind

**Comment by doodlewind** [comment]
If we are still supporting Next, it looks fine with GitHub CI and Vercel build config both enabled, which is our current state.
